bin/http_server.re

```diff
@@ -179,6 +179,15 @@ let handle_ticket_balance =
 let node = {
   let folder = Sys.argv[1];
   let.await identity = Files.Identity.read(~file=folder ++ "/identity.json");
+  let.await trusted_validators =
+    Files.Validators.read(~file=folder ++ "/trusted-validators.json")
```

Comment:
It's not trusted validators, is trusted-new-validators, this is a set of validators who are authorized to join the network once.

Reply:
Also I would ask you to make another file on `Files`

Reply:
what does it mean for a validator to be authorized to join the network once? do validators join the network multiple times?

Reply:
@anchpop Before adding/removing a validator, existing validators decided off-chain who is getting booted/added. Each one of them instruct their respective nodes, "Hey, if you see on-chain operation saying add this validator, and it happens to be present in `trusted_validators` accept it. Else reject"

(apologies for anthromorphising things :D )